server/container/cli-integration/src/test/java/org/apache/james/cli/SieveCommandsIntegrationTest.java

```diff
@@ -121,4 +121,18 @@ public void removeSieveQuotaShouldWork() throws Exception {
         sieveProbe.getSieveQuota();
     }
 
+    @Test
+    public void addActiveSieveScript() throws Exception {
+        String script = "require \"fileinto\";\n" +
+                "\n" +
+                "fileinto \"INBOX.any\";";
+
+        ServerCmd.doMain(new String[] {"-h", "127.0.0.1", "-p", "9999", "ADDSIEVESCRIPT", USER, "myscript.sieve", script});
```

Comment:
I would prefer to pass the path of the script to import rather than the script directly...

 - This would be  more consistant with ImportEml command

Maybe ADDSIEVESCRIPT USER path/to/myscript.sieve

(You can append .sieve at the end if the extension is not the good one BTW)